website/templates/project/project_base.mako

```diff
@@ -214,6 +214,7 @@
             registrationMetaSchemas: ${ node['registered_schemas'] | sjson, n },
             registrationMetaData: ${ node['registered_meta'] | sjson, n },
             contributors: ${ node['contributors'] | sjson, n },
+            viewOnlyLink: ${ node['link'] | sjson, n }
```

Comment:
Why does this need to be added to contextVars? Can you just access it from the context vars?